internal/server/v0/triple/golden/get_triples/stat_var.json

```diff
@@ -1129,149 +1129,6 @@
         "StatisticalVariable"
       ],
       "provenanceId": "dc/d7tbsb1"
-    },
-    {
-      "subjectId": "dc/06w4nhpt1vz26",
```

Comment:
Not sure about this too

Reply:
Ditto. This is Count_Product_MobileCellularSubscription_AsFractionOf_Count_Person